modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/ApplicationPolicyDeploymentTestCase.java

```diff
@@ -83,6 +90,12 @@
 
   private static File simpleExtensionJarFile;
   private static File withErrorDeclarationExtensionJarFile;
+
+  @Parameterized.Parameters(name = "Parallel: {0}")
+  public static List<Boolean> params() {
```

Comment:
remove this

Reply:
Don't think we should remove. We are running policy tests twice for parallel and non-parallel deployment when that does not affect policies at all.

Reply:
can you add that as a comment in code?